RLTest/redis_std.py

```diff
@@ -242,7 +242,7 @@ def createCmdArgs(self, role):
 
         if self._getRedisVersion() > 70000:
             if self.enableDebugCommand:
-                cmdArgs += ['--enable-debug-command', 'yes']
+                cmdArgs += ['--enable-debug-command', 'yes', '--enable-module-command', 'yes']
```

Comment:
Did we concluded what was the reason we did not do the same for `enableProtectedConfigs`?